components/grade-table.js

```diff
@@ -0,0 +1,24 @@
+class GradeTable {
+  constructor(tableElement) {
+    this.tableElement = tableElement;
+  }
+  updateGrades(grades) {
+    var tBody = this.tableElement.querySelector('tbody');
+    console.log(grades);
```

Comment:
Make sure to remove `console.log`s once you are done with them and before making pull requests to master. Production code shouldn't have `console.log`s in it, the only exception being `console.error` for errors.